### PR TITLE
Update rust doc - DVM part 

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

File: frame/dvm/src/account_basic.rs [hunk 1]
```diff
@@ -16,6 +16,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
+//! The module that manages the basic account info display in dvm.
```
Comment (AurevoirXavier) [inline frame/dvm/src/account_basic.rs:19]: pallet

Comment (AsceticBear) [reply frame/dvm/src/account_basic.rs]: `account_basic` is just a mod, not a pallet.

Comment (AurevoirXavier) [reply frame/dvm/src/account_basic.rs]: How about rust module

rust/runtime module make people confuse

**Comment by AurevoirXavier** [comment]
Please don't merge before I review ; )

File: frame/dvm-dynamic-fee/src/lib.rs [hunk 2]
```diff
@@ -32,25 +36,26 @@ pub mod pallet {
 	// --- darwinia-network ---
 	use darwinia_evm::FeeCalculator;
 
-	pub type InherentType = U256;
+	pub(super) type InherentType = U256;
 
-	pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"dynfee0_";
+	pub(super) const INHERENT_IDENTIFIER: InherentIdentifier = *b"dynfee0_";
```
Comment (AurevoirXavier) [inline frame/dvm-dynamic-fee/src/lib.rs:41]: Any idea that you don't want to expose these?

Comment (AsceticBear) [reply frame/dvm-dynamic-fee/src/lib.rs:41]: I remember that I have searched other pallets in substrate frames that use inherent design and following their doc style.

Comment (AurevoirXavier) [reply frame/dvm-dynamic-fee/src/lib.rs:41]: Any example?

Comment (AsceticBear) [reply frame/dvm-dynamic-fee/src/lib.rs:41]: https://crates.parity.io/pallet_timestamp/index.html

**Pull request merged**